README.md

```diff
@@ -1,12 +1,16 @@
 # AICSImageIO
 
+> [!WARNING]  
+> Deprecated in favor of replacement tool `bioio` ([see here for migration guide](https://bioio-devs.github.io/bioio/MIGRATION.html))
```

Comment:
```suggestion
> AICSImageIO is now in maintenance mode only.  Please take a look at its compatible successor [`bioio`](https://github.com/bioio-devs/bioio) ([see here for migration guide](https://bioio-devs.github.io/bioio/MIGRATION.html))
```

Reply:
If you want to be fancy, I sometimes see people include: ⚠️⚠️ around it or some other emoji to grab yet more attention.